docs/docs/09-commands-and-tools/03-hot-reload.md

```diff
@@ -82,6 +82,20 @@ sequenceDiagram
     deactivate templ_proxy
 ```
 
+### Triggering hot reload from outside `templ generate --watch`
+
+If you want to trigger a hot reload from outside `templ generate --watch` (e.g. if you're using `air`, `wgo` or another tool to build, but you want to use the templ hot reload proxy), you can use the `--notify-proxy` argument.
+
+```shell
+templ generate --notify-proxy
+```
+
+This will default to the default templ proxy address of `localhost:7331`, but can be changed with the `--proxybind` and `--proxyport` arguments.
+
+```shell
+templ generate --notify-proxy --proxybind="localhost" --proxyport="8080"
+```
+
```

Comment:
Great work, this is a nice addition to the templ CLI, will be using this once it's released.
Maybe even replace --watch?

I was thinking that a change to the explanation, to make it more clear, could help this feature be more utilized when it's released.

By specifying that --notify-proxy takes priority over other flags, and will be exec just once, can help developers better understand the usage for this cmd.
Also, the first time I read this text, it felt like I should add --notify-proxy as a flag to the templ generate --watch cmd, which would end in an unexpected outcome. (perhaps just my thinking tho :P )

Could also mention that --notify-proxy only works if --proxy is used. You kinda already does this and it's somewhat self explanatory, but one extra sentence could clear up any misunderstandings?

PS: some of this can maybe also be applied to the --notify-proxy CLI help text.

This was just some thoughts on my part.